src/stores/air_to_ground/missiles/shrike.md

```diff
@@ -194,29 +194,39 @@ release button. The Shrike will fire within one second after pressing the bomb r
 aware that the Shrike will have the least amount of range in direct mode and should be fired very
 near to the target. A dive angle of 20 degree or more is recommended.
 
-### Steering Profiles
+### Attack Profiles
 
-The weapon itself can be configured for two different steering profiles called _Loft_
-and _Direct_ (not to be confused with the attack profiles available to employ them).
+The missile itself can be configured for two different attack profiles called _Loft Attack_ and
+_Direct Attack_ (not to be confused with the employment profiles available to employ them).
 
-These control when the missiles control surfaces can be actuated after launch.
+These control when the missile's guidance will initiate after launch.
 
-#### Loft Steering
+> 💡 Selection of the improper attack profile might cause the AGM-45 to miss the target due to lack
+> of energy (ex: the bang-bang guidance activating too early) or overshooting the target (ex:
+> guidance doesn't enable in time).
 
-When firing the missile, control surfaces are locked until the missile descended below the
-initial launch altitude.
+#### Loft Attack
 
-This prevents unnecessary loss of energy due to the bang-bang control surfaces of the weapon.
+With "Loft Attack", the guidance will initiate when the missile's barometric altimeter detects a
+pressure increase of 1 psi AND the missile's barometric altimeter senses that the missile is below
+18,000ft MSL.
```

Comment:
Minor, but from an editorial perspective I am not a fan of using capslock `AND` to separate the two conditions visually.
Instead I would suggest to use bullet points. So something like

> With _Loft Attack_, the guidance will initiate when
> * the missile's barometric altimeter detects a
pressure increase of 1 psi and
> * the missile's barometric altimeter senses that the missile is below
18,000ft MSL.